numalogic/config/factory.py

```diff
@@ -146,6 +147,7 @@ class ModelFactory(_ObjectFactory):
         "TransformerAE": TransformerAE,
         "SparseTransformerAE": SparseTransformerAE,
         "Conv1dVAE": Conv1dVAE,
+        "VanillaICAE": VanillaICAE,
```

Comment:
This name looks little odd.

Reply:
Will change later once Miro verifies the usage